keyboards/togkey/pad_pocket/config.h

```diff
@@ -0,0 +1,7 @@
+// Copyright 2025 TogKey
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+//Tap Dance
+#define TAPPING_TERM 250
```

Comment:
This should be migrated to `keyboard.json`, if you feel it is necessary. Regardless, this file should be removed.